app/views/directory_objects/departmentPlacards.html.erb

```diff
@@ -1,48 +1,133 @@
+			toggleBGImg = 1;
+			break;
 
-	<body>
-			<!-- This is the placard background -->
+			case 1:	
+			newBackground = "/assets/placard/Modern1.jpg";
+			toggleBGImg = 2;
+			break;
+
+			case 2:
+			newBackground = "/assets/placard/nature.jpg";
+			toggleBGImg = 3;
+			break;
+
+			case 3:
+			newBackground = " ";
+			toggleBGImg = 0;
+			break;
+				
+			default:
+			newBackground = "/assets/placard/back1.jpg";
+			toggleBGImg = 1;
+			break;
+			}
+
+		var i;
+		for (i = 0; i < background.length; i++) {
+			background[i].setAttribute("src", newBackground);
+		}
+
+	}
+			
+	</script>
+
+
+    <!-- This is the placard background -->
     <% @results.each do |person| %>
-  		<div class="backgroundImg">
-        <%= image_tag("placard/back1.jpg", id: "bgImage", alt: "back", class: "bgstyle") %>
+      <div class="backgroundImg">
+        <%= image_tag("placard/back1.jpg", id: "bgImage", alt: " ", class: "bgstyle") %>
 
-  			<!-- This is the name field -->
-  			<div id="nameField" class="name"> <%= person[:name] %>
+        <!-- This is the name field -->
+        <div id="nameField" class="name"> <%= person[:name] %>
 
-  			<!--Extra (watch for div enclosure-->
-  			<div id="titleField" class="title"><%= person[:title] %></div>
+        <!--Extra (watch for div enclosure)-->
+        <div id="titleField" class="title"><%= person[:title] %></div>
 
-  			</div>
+        </div>
 
-  			<!-- This is the QR image-->
-  			<div class="qr">
+        <!-- This is the QR image-->
+        <div class="qr">
             <%= image_tag(person[:qrLink], alt: "qrCode", class: "qrstyle", id: "qrImage") %>
-  			</div>
+        </div>
 
-  			<!-- ===============EXTRA=================-->
-  			<div class="logo">
-          <!-- image_tag("icon.png", size: "16x10", alt: "Edit Entry") -->
-  					<%= image_tag("placard/davislogo.png", alt: "logoCode", class: "logostyle", id: "logoImage") %>
-  			</div>
+        <!-- ===============EXTRA=================-->
+	<!-- This is the school logo -->
+        <div class="logo">
+            <%= image_tag("placard/davislogo.png", alt: "logoCode", class: "logostyle", id: "logoImage") %>
+        </div>
 
+        <!-- This is the department name field -->
+        <div id="depNameField" class="depname"><%= person[:department] %></div>
 
-  			<!-- This is the department name field -->
-  			<div id="depNameField" class="depname"><%= person[:department] %></div>
+	<!-- This is the email address-->
+	<div id="emailField" class="email">
+	<%= person[:email] %>
+	</div>
 
-  			<!-- =====================================-->
-  		</div>
+        <!-- =====================================-->
+      </div>
+	<br>
     <% end %>
 
-		<p> Cut on dotted line </p>
-		<p> Thanks for participating in the new and improved wayfinding system! </p>
+	<!-- This is where the user can change the background images-->
+	<button onclick="changeBgImage()">Change Background Images</button>
+
+	<br>
```

Comment:
Use paragraph, div, or span tags with the appropriate padding and margin settings instead of <br>. You can emulate three BRs, e.g. padding-bottom: 3em;